packages/lib/convertToNewDurationType.ts

```diff
@@ -0,0 +1,35 @@
+export const MINUTES_IN_HOUR = 60;
+export const MINUTES_IN_DAY = 1440;
+export const HOURS_IN_DAY = 24;
+
+export default function convertToNewDurationType(prevType: string, newType: string, prevValue: number) {
```

Comment:
Shall we create a union for `minutes, hours, days`? This way can't make that typo anymore! 😁

Reply:
@JeroenReumkens This is a great idea! ill add this now! Thank you so much